bundles/org.openhab.binding.mybmw/.gitignore

```diff
@@ -0,0 +1 @@
+jacoco.exec
```

Comment:
why should I remove the .gitignore file? It is a standard file to tell git to not commit pre-defined files to the repository, see https://git-scm.com/docs/gitignore

Reply:
.gitignore is a perfect usuable file. No need to explain that :-) It is just not common for atleast the addons repository to have this file added to the repository. From the 400+ bundels i only see 2 occurences for specific persistence bundles.

Reply:
OK I can remove it. I just then have to be careful when I commit my changes, but they should not be too many right now as the binding seems to be running without major issues since February. I just had to fix some conflicts due to core library changes since then

Reply:
done